modules/ballerina-http/src/main/ballerina/ballerina/net/http/httplib.bal

```diff
@@ -0,0 +1,25 @@
+package ballerina.net.http;
+
+const string CONTINUE_100 = "100-continue";
+
+@Description { value:"This function can be used to check whether the client expects a 100-continue response."}
+@Param { value:"req: A request struct" }
+@Return { value:"Returns true if the client expects a 100-continue response. If not, returns false." }
+public function <Request req> expects100Continue() (boolean) {
+    string expectHeader = req.getHeader("Expect");
+
+    if (expectHeader == CONTINUE_100) {
+        return true;
+    }
+
```

Comment:
Shall we remove this new line too? Looks inconsistent with other contents in the file